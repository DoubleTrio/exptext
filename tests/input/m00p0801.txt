message_SetFace(ACTOR_NPC_DAGUTORIO, FACE_NORMAL, FACE_POS_STANDARD);
message_Talk({
    english=" Gwah![K] Gwargh![K] Gwagagah! [hero], [c_kind:NPC_ZUBATTO]",
    french=" Gwah![K] Gwaaaah![K]\nGwaaaaahgaaaaah! [hero], [c_kind:NPC_ZUBATTO]",
    german=" Gwah![K] Gwargh![K] Gwagagah! [hero], [c_kind:NPC_ZUBATTO]",
    italian=" Uh![K] Argh![K] Aaaargh! [hero], [c_kind:NPC_ZUBATTO]",
    spanish=" ¡Ah![K] ¡Aaah![K] ¡Aaaaaaaah! [hero], [c_kind:NPC_ZUBATTO]",
});