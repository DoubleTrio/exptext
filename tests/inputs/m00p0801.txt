def 0 {
    with (actor ACTOR_ATTENDANT1) {
        SetAnimation(2);
    }
    with (actor ACTOR_PLAYER) {
        SetAnimation(2);
    }
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_PLAYER);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    with (actor ACTOR_PLAYER) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    switch ( ProcessSpecial(PROCESS_SPECIAL_IS_TEAM_SETUP_PARTNER_AND_HERO_ONLY, 0, 0) ) {
        case 1:
            §label_0;
            with (actor ACTOR_ATTENDANT1) {
                Turn2DirectionLives(4, 10, ACTOR_PLAYER);
            }
            WaitExecuteLives(ACTOR_ATTENDANT1);
            message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" Hey, [hero].",
                    }
                case 2:
                    {
                        english=" Listen, [hero].",
                    }
                default:
                    {
                        english=" Say, [hero].",
                    }
            }
            CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
            with (actor ACTOR_PLAYER) {
                Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
            }
            WaitExecuteLives(ACTOR_PLAYER);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nought to lead, [hero].",
                    }
                case 2:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nshould be the leader, [hero].",
                    }
                default:
                    {
                        english=" When we go to [CS:P]Mt. Travail[CR], you\nshould go as the leader, [hero].",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" After all, that\'s the way we\'ve\nalways done it for important missions.",
                    }
                case 2:
                    {
                        english=" After all, that\'s the way we\'ve\nalways handled things on important missions.",
                    }
                default:
                    {
                        english=" After all, we\'ve always gone on\nour important missions that way.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" So I was thinking that you ought\nto be the leader again, [hero]...[K]\nWhat do you say?",
                    }
                case 2:
                    {
                        english=" That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?",
                    }
                default:
                    {
                        english=" That\'s why I think you should\nbe the leader again, [hero]...[K]\nWhat do you say?",
                    }
            }
            switch ( message_SwitchMenu(0, 1) ) {
                case menu({
                    english="Accept leadership",
                }):
                default:
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    switch ( ProcessSpecial(PROCESS_SPECIAL_SET_TEAM_SETUP_HERO_AND_PARTNER_ONLY, 0, 0) ) { }
                    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Cool![K] That\'s it, then! You\'re the\nleader, [hero].",
                            }
                        case 2:
                            {
                                english=" OK![K] That\'s settled. You\'re the\nleader, [hero].",
                            }
                        default:
                            {
                                english=" OK![K] You\'re the team leader,\n[hero]!",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Let\'s get rolling to [CS:P]Mt. Travail[CR]!",
                            }
                        case 2:
                            {
                                english=" Let\'s set out for [CS:P]Mt. Travail[CR]!",
                            }
                        default:
                            {
                                english=" We should get moving to\n[CS:P]Mt. Travail[CR]!",
                            }
                    }
                    message_Close();
                    end;
                case menu({
                    english="Refuse",
                }):
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    se_Play(8968);
                    with (actor ACTOR_ATTENDANT1) {
                        SetEffect(EFFECT_SHOCKED, 3);
                    }
                    with (actor ACTOR_ATTENDANT1) {
                        WaitEffect();
                    }
                    WaitExecuteLives(ACTOR_ATTENDANT1);
                    message_SetFace(ACTOR_ATTENDANT1, FACE_SURPRISED, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Whoa![K] Hang on, we\'re going\nwith me as leader?",
                            }
                        case 2:
                            {
                                english=" What?[K] Wait, so I\'m going as\nthe leader?",
                            }
                        default:
                            {
                                english=" What?[K] So I\'m going to go as\nthe leader?",
                            }
                    }
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    message_SetFace(ACTOR_ATTENDANT1, FACE_WORRIED, FACE_POS_STANDARD);
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Hmm...[K] I don\'t feel confident\nabout doing that...",
                            }
                        case 2:
                            {
                                english=" Hmm...[K] I don\'t have the\nconfidence to do that...",
                            }
                        default:
                            {
                                english=" Hmm...[K] I don\'t feel confident\nenough to be the leader...",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...",
                            }
                        case 2:
                            {
                                english=" If we\'re going to [CS:P]Mt. Travail[CR],\nyou should be the leader, [hero]...",
                            }
                        default:
                            {
                                english=" If we\'re still going to go to\n[CS:P]Mt. Travail[CR], you should lead, [hero]...",
                            }
                    }
                    message_Close();
                    end;
            }
            break;
        default:
            message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_STANDARD);
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" Hey, [hero].[K] How about if\nwe just go as a pair, you and I?",
                    }
                case 2:
                    {
                        english=" Listen, [hero].[K] What\nwould you say if I suggested that just you and\nI go?",
                    }
                default:
                    {
                        english=" Say, [hero].[K] Do you think\nmaybe we should just go as a pair, us two?",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" If a whole bunch of us went,\nit\'d probably upset [CS:N]Drowzee[CR] beyond the point\nof persuasion...",
                    }
                case 2:
                    {
                        english=" If a whole group went, [CS:N]Drowzee[CR]\nwould probably get upset. You can forget about\npersuading him then.",
                    }
                default:
                    {
                        english=" If a whole lot of us go, [CS:N]Drowzee[CR]\nwill probably get upset with us. He would be\nimpossible to persuade like that.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" But going alone won\'t do it,\neither. One Pokémon won\'t be convincing\nenough.",
                    }
                case 2:
                    {
                        english=" But going alone won\'t be enough.\nI don\'t think it would be persuasive enough to\nget the point across.",
                    }
                default:
                    {
                        english=" But I think one Pokémon alone\nwon\'t be persuasive enough.",
                    }
            }
            message_SwitchTalk ($PARTNER_TALK_KIND) {
                case 1:
                    {
                        english=" So how about just we two go?",
                    }
                case 2:
                    {
                        english=" That\'s why I suggested the two\nof us go.",
                    }
                default:
                    {
                        english=" So what do you think? Should\njust the two of us go?",
                    }
            }
            switch ( message_SwitchMenu(0, 1) ) {
                case menu({
                    english="Go as a pair",
                }):
                default:
                    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
                    switch ( ProcessSpecial(PROCESS_SPECIAL_SET_TEAM_SETUP_HERO_AND_PARTNER_ONLY, 0, 0) ) { }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" OK![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!",
                            }
                        case 2:
                            {
                                english=" That\'s settled![K] Let\'s get going to\n[CS:P]Mt. Travail[CR]!",
                            }
                        default:
                            {
                                english=" OK![K] Let\'s go to [CS:P]Mt. Travail[CR]!",
                            }
                    }
                    message_SwitchTalk ($PARTNER_TALK_KIND) {
                        case 1:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                        case 2:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                        default:
                            {
                                english=" Let\'s do our best, [hero]!",
                            }
                    }
                    message_Close();
                    end;
                case menu({
                    english="Don\'t go",
                }):
                    message_Close();
                    end;
            }
            break;
    }
}
