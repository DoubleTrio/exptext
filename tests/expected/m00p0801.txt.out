[Kunan](1): Hey, Yugure.
[Kunan](2): Listen, Yugure.
[Kunan](3): Say, Yugure.
[Kunan](1): When we go to Mt. Travail, you ought to lead, Yugure.
[Kunan](2): When we go to Mt. Travail, you should be the leader, Yugure.
[Kunan](3): When we go to Mt. Travail, you should go as the leader, Yugure.
[Kunan](1): After all, that's the way we've always done it for important missions.
[Kunan](2): After all, that's the way we've always handled things on important missions.
[Kunan](3): After all, we've always gone on our important missions that way.
[Kunan](1): So I was thinking that you ought to be the leader again, Yugure... What do you say?
[Kunan](2): That's why I think you should be the leader again, Yugure... What do you say?
[Kunan](3): That's why I think you should be the leader again, Yugure... What do you say?
[case_menu]: Accept leadership
[Kunan](1): Cool! That's it, then! You're the leader, Yugure.
[Kunan](2): OK! That's settled. You're the leader, Yugure.
[Kunan](3): OK! You're the team leader, Yugure!
[Kunan](1): Let's get rolling to Mt. Travail!
[Kunan](2): Let's set out for Mt. Travail!
[Kunan](3): We should get moving to Mt. Travail!
[case_menu]: Refuse
[Kunan](1): Whoa! Hang on, we're going with me as leader?
[Kunan](2): What? Wait, so I'm going as the leader?
[Kunan](3): What? So I'm going to go as the leader?
[Kunan](1): Hmm... I don't feel confident about doing that...
[Kunan](2): Hmm... I don't have the confidence to do that...
[Kunan](3): Hmm... I don't feel confident enough to be the leader...
[Kunan](1): If we're going to Mt. Travail, you should be the leader, Yugure...
[Kunan](2): If we're going to Mt. Travail, you should be the leader, Yugure...
[Kunan](3): If we're still going to go to Mt. Travail, you should lead, Yugure...
[Kunan](1): Hey, Yugure. How about if we just go as a pair, you and I?
[Kunan](2): Listen, Yugure. What would you say if I suggested that just you and I go?
[Kunan](3): Say, Yugure. Do you think maybe we should just go as a pair, us two?
[Kunan](1): If a whole bunch of us went, it'd probably upset Drowzee beyond the point of persuasion...
[Kunan](2): If a whole group went, Drowzee would probably get upset. You can forget about persuading him then.
[Kunan](3): If a whole lot of us go, Drowzee will probably get upset with us. He would be impossible to persuade like that.
[Kunan](1): But going alone won't do it, either. One Pokémon won't be convincing enough.
[Kunan](2): But going alone won't be enough. I don't think it would be persuasive enough to get the point across.
[Kunan](3): But I think one Pokémon alone won't be persuasive enough.
[Kunan](1): So how about just we two go?
[Kunan](2): That's why I suggested the two of us go.
[Kunan](3): So what do you think? Should just the two of us go?
[case_menu]: Go as a pair
[Kunan](1): OK! Let's get going to Mt. Travail!
[Kunan](2): That's settled! Let's get going to Mt. Travail!
[Kunan](3): OK! Let's go to Mt. Travail!
[Kunan](1): Let's do our best, Yugure!
[Kunan](2): Let's do our best, Yugure!
[Kunan](3): Let's do our best, Yugure!
[case_menu]: Don't go
